<launch>

	<arg name="legacy_packets" default="false" />
	<param name="use_legacy_packets" type="boolean" value="$(arg legacy_packets)" />

	<!-- <arg name="robot_output_target" default="serial" /> -->
	<param name="robot_output_target" type="string" value="$(arg robot_output_target)" />

	<arg name="robot_output_target" />
	<param name="robot_output_target" type="string" value="$(arg robot_output_target)" />

	<param name="role_iterations_per_second" type="int" value="60" />
	
	

	<node pkg="roboteam_robothub" type="roboteam_robothub" name="robothub" output="screen" />

</launch>